{"name": "test_enterDataInForm", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <SeleniumFramework.pages.ContactFormPage.ContactForm object at 0x0000011EA5823D90>\nlocatorValue = 'e666mail', locatorType = 'id'\n\n    def waitForElement(self, locatorValue, locatorType=\"id\"):\n        webElement = None\n        try:\n            locatorType = locatorType.lower()\n            locatorByType = self.getLocatorType(locatorType)\n            wait = WebDriverWait(self.driver, 25, poll_frequency=1,\n                                 ignored_exceptions=[ElementNotVisibleException, NoSuchElementException])\n>           webElement = wait.until(ec.presence_of_element_located((locatorByType, locatorValue)))\n\n..\\base\\BasePage.py:69: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"0961147df4399bbbce1c2dd0cbda97ab\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x0000011EA583B4C0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       \tGetHandleVerifier [0x00007FF70FA552A2+57122]\nE       \t(No symbol) [0x00007FF70F9CEA92]\nE       \t(No symbol) [0x00007FF70F89E3AB]\nE       \t(No symbol) [0x00007FF70F8D7D3E]\nE       \t(No symbol) [0x00007FF70F8D7E2C]\nE       \t(No symbol) [0x00007FF70F910B67]\nE       \t(No symbol) [0x00007FF70F8F701F]\nE       \t(No symbol) [0x00007FF70F90EB82]\nE       \t(No symbol) [0x00007FF70F8F6DB3]\nE       \t(No symbol) [0x00007FF70F8CD2B1]\nE       \t(No symbol) [0x00007FF70F8CE494]\nE       \tGetHandleVerifier [0x00007FF70FCFEF82+2849794]\nE       \tGetHandleVerifier [0x00007FF70FD51D24+3189156]\nE       \tGetHandleVerifier [0x00007FF70FD4ACAF+3160367]\nE       \tGetHandleVerifier [0x00007FF70FAE6D06+653702]\nE       \t(No symbol) [0x00007FF70F9DA208]\nE       \t(No symbol) [0x00007FF70F9D62C4]\nE       \t(No symbol) [0x00007FF70F9D63F6]\nE       \t(No symbol) [0x00007FF70F9C67A3]\nE       \tBaseThreadInitThunk [0x00007FFC47597344+20]\nE       \tRtlUserThreadStart [0x00007FFC486C26B1+33]\n\n..\\..\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <SeleniumFramework.pages.ContactFormPage.ContactForm object at 0x0000011EA5823D90>\ntext = 'example@email.com', locatorValue = 'e666mail', locatorType = 'id'\n\n    def sendText(self, text, locatorValue, locatorType=\"id\"):\n        try:\n            locatorType = locatorType.lower()\n>           webElement = self.waitForElement(locatorValue, locatorType)\n\n..\\base\\BasePage.py:95: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <SeleniumFramework.pages.ContactFormPage.ContactForm object at 0x0000011EA5823D90>\nlocatorValue = 'e666mail', locatorType = 'id'\n\n    def waitForElement(self, locatorValue, locatorType=\"id\"):\n        webElement = None\n        try:\n            locatorType = locatorType.lower()\n            locatorByType = self.getLocatorType(locatorType)\n            wait = WebDriverWait(self.driver, 25, poll_frequency=1,\n                                 ignored_exceptions=[ElementNotVisibleException, NoSuchElementException])\n            webElement = wait.until(ec.presence_of_element_located((locatorByType, locatorValue)))\n            self.log.info(\"WebElement found with locator value: \" + locatorValue + \" using locatorType: \" + locatorType)\n        except:\n            self.log.error(\n                \"WebElement not found with locator value: \" + locatorValue + \" using locatorType: \" + locatorType)\n            print_stack()\n            self.takeScreenshot(locatorType)\n>           assert False\nE           AssertionError\n\n..\\base\\BasePage.py:76: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <SeleniumFramework.tests.TestContactForm.ContactFormTest testMethod=test_enterDataInForm>\n\n    @pytest.mark.run(order=2)\n    def test_enterDataInForm(self):\n        self.cf.enterName()\n>       self.cf.enterEmail()\n\nTestContactForm.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\pages\\ContactFormPage.py:41: in enterEmail\n    self.sendText('example@email.com', self._enterEmail, 'id')\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <SeleniumFramework.pages.ContactFormPage.ContactForm object at 0x0000011EA5823D90>\ntext = 'example@email.com', locatorValue = 'e666mail', locatorType = 'id'\n\n    def sendText(self, text, locatorValue, locatorType=\"id\"):\n        try:\n            locatorType = locatorType.lower()\n            webElement = self.waitForElement(locatorValue, locatorType)\n            webElement.send_keys(text)\n            self.log.info(\"Sent the text: \" + text + \" in WebElement with locator value: \" + locatorValue\n                          + \" using locatorType: \" + locatorType)\n        except:\n            self.log.error(\n                \"Unable to Sent the text: \" + text + \" in WebElement with locator value: \" + locatorValue\n                + \"using locatorType: \" + locatorType)\n            print_stack()\n            self.takeScreenshot(locatorType)\n>           assert False\nE           AssertionError\n\n..\\base\\BasePage.py:105: AssertionError"}, "steps": [{"name": "Entered Name", "status": "passed", "start": 1694987092488, "stop": 1694987092488}], "attachments": [{"name": "id", "source": "01f83eb4-8b97-4aeb-9bcf-059ca5513ccf-attachment.png", "type": "image/png"}, {"name": "id", "source": "1cc86f6a-6248-4484-b1df-1e1515a98c6e-attachment.png", "type": "image/png"}, {"name": "log", "source": "da04bd47-016a-4af2-986c-cb7c2730bd14-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "738c7acc-0279-4f61-8b5d-99bf5da270ed-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "bcac1c5e-0262-4dbe-9699-30b953eab209-attachment.txt", "type": "text/plain"}], "start": 1694987092407, "stop": 1694987118237, "uuid": "e1bc3f3c-3d11-456e-9589-6d60571d65c9", "historyId": "21daca012456f37e07609114a6b1953c", "testCaseId": "21daca012456f37e07609114a6b1953c", "fullName": "TestContactForm.ContactFormTest#test_enterDataInForm", "labels": [{"name": "tag", "value": "run(order=2)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup_driver', 'setup_method')"}, {"name": "suite", "value": "TestContactForm"}, {"name": "subSuite", "value": "ContactFormTest"}, {"name": "host", "value": "LAPTOP-SFQ2DO00"}, {"name": "thread", "value": "21868-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestContactForm"}]}